main/precompiled/selector.zkasm

```diff
@@ -10,11 +10,11 @@ INCLUDE "end.zkasm"
  */
 selectorPrecompiled:
     A - 2               :JMPN(funcECRECOVER)
-    A - 3               :JMPN(handleGas)  ;:JMPN(SHA256)
-    A - 4               :JMPN(handleGas)  ;:JMPN(RIPEMD160)
+    A - 3               :JMPN(readCode)  ;:JMPN(SHA256)
+    A - 4               :JMPN(readCode)  ;:JMPN(RIPEMD160)
     A - 5               :JMPN(IDENTITY)
     A - 6               :JMPN(MODEXP)
-    A - 7               :JMPN(handleGas) ;:JMPN(ECADD)
-    A - 8               :JMPN(handleGas) ;:JMPN(ECMUL)
-    A - 9               :JMPN(handleGas) ;:JMPN(ECPAIRING)
-    A - 10              :JMPN(handleGas) ;:JMPN(BLAKE2F)
\ No newline at end of file
+    A - 7               :JMPN(readCode) ;:JMPN(ECADD)
+    A - 8               :JMPN(readCode) ;:JMPN(ECMUL)
+    A - 9               :JMPN(readCode) ;:JMPN(ECPAIRING)
+    A - 10              :JMPN(readCode) ;:JMPN(BLAKE2F)
```

Comment:
Should be `callContract ` instead of :JMPN(readCode) bc then no value will be transfered